.github/workflows/linux_conditional_compilation.yml

```diff
@@ -66,75 +74,52 @@ jobs:
 
       - name: Install build dependencies
         run: |
-          sudo -E ${{ env.OPENVINO_REPO }}/install_build_dependencies.sh
-          sudo -E apt update
-          sudo -E apt --assume-yes install openjdk-11-jdk libbz2-dev clang unzip libpugixml-dev libtbb-dev intel-opencl-icd ocl-icd-opencl-dev opencl-headers
+          bash ${OPENVINO_REPO}/install_build_dependencies.sh
+          # unzip - to download ninja
+          apt install --assume-yes --no-install-recommends unzip
 
-          wget https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-linux.zip
+          wget https://github.com/ninja-build/ninja/releases/download/v1.11.1/ninja-linux.zip
           unzip ninja-linux.zip
```

Comment:
ninja is installed as a part of ` bash ${OPENVINO_REPO}/install_build_dependencies.sh`